internal/policies/privilege/privilege.go

```diff
@@ -243,10 +299,94 @@ func splitAndNormalizeUsersAndGroups(ctx context.Context, v string) []string {
 	return elems
 }
 
-// getSystemPolkitAdminIdentities returns the list of configured system polkit admins as a string.
+func (m *Manager) getSystemPolkitAdminIdentities(ctx context.Context, policyKitDir string, oldPolkit bool) (adminIdentities string, err error) {
+	if oldPolkit {
+		return polkitAdminIdentitiesFromConf(ctx, policyKitDir)
+	}
+	return polkitAdminIdentitiesFromRules(ctx, []string{
+		policyKitDir,
+		m.policyKitSystemDir,
+	})
+}
+
+func polkitAdminIdentitiesFromRules(ctx context.Context, rulesDirPaths []string) (adminIdentities string, err error) {
```

Comment:
No comment either :) Also, it can be beneficial to explain the logic: we take the first file in ascii order matching blablabla, independent of them being in /etc or /usr.